src/main.js

```diff
@@ -95,24 +62,23 @@ function sum(a) {
 console.log(sum(5)(2).toString());
 console.log(sum(3)(7).toString());
 
-const colors = ['magenta', 'cyan', 'firebrick', 'springgreen', 'skyblue'];
-
 const p1 = document.getElementById('text1');
 const p2 = document.getElementById('text2');
 const p3 = document.getElementById('text3');
 const paragraphs = [p1, p2, p3];
-for (let i = 0; i < paragraphs.length; i++){
-  paragraphs[i].addEventListener("click", changeColor(paragraphs[i]))
-}
-function changeColor(elem) {
-  let counter = 0;
-  return function () {
-    if (counter === colors.length) {
-      counter = 0;
-    }
-    elem.style.color = colors[counter];
-    counter++
-  }
+for (let i = 0; i < paragraphs.length; i++) {
+  paragraphs[i].addEventListener('click', changeColor(paragraphs[i]));
 }
+import { changeColor } from './Functions(lesson4)/changeColor.js';
+//lesson5
+import { changeDate } from './String(lesson5)/changeData.js';
+console.log(changeDate('2020-11-28'));
+
+import { stringSearch } from './String(lesson5)/stringSearch.js';
+
+console.log(stringSearch('Berlin'));
+console.log(stringSearch('Russia'));
+console.log(stringSearch('Villa Kunerad'));
+console.log(stringSearch('Belarus'));
 
 module.exports = changeColor;
```

Comment:
<img width="1216" alt="Screenshot 2022-05-30 at 20 42 42" src="https://user-images.githubusercontent.com/25488411/171033633-a7cf9401-dbf8-46f2-a717-40b8b4f39c19.png">
you can't use 2 types of export
if you see errors in console, try to fix them before PR